src/main/java/ch/naviqore/raptor/RaptorBuilder.java

```diff
@@ -20,19 +18,30 @@
  *
  * @author munterfi
  */
-@NoArgsConstructor(access = AccessLevel.PACKAGE)
 @Log4j2
 public class RaptorBuilder {
 
+    private final static int DEFAULT_SAME_STATION_TRANSFER_TIME = 120;
+
+    private final int defaultSameStationTransferTime;
     private final Map<String, Integer> stops = new HashMap<>();
     private final Map<String, RouteBuilder> routeBuilders = new HashMap<>();
     private final Map<String, List<Transfer>> transfers = new HashMap<>();
+    private final Map<String, Integer> sameStationTransfers = new HashMap<>();
     private final Map<String, Set<String>> stopRoutes = new HashMap<>();
 
     int stopTimeSize = 0;
     int routeStopSize = 0;
     int transferSize = 0;
 
+    RaptorBuilder() {
+        this(DEFAULT_SAME_STATION_TRANSFER_TIME);
```

Comment:
Same here, then only one constructor is needed.

Reply:
Removed in cc10db6